### ISSUE TITLE
Export partial Maya joint hierarchy to USD as Skeleton prims

### ISSUE THREAD

**Issue by NickWu** [Autodesk/maya-usd#3596]
**Is your feature request related to a problem? Please describe.**
Hi, we couldn't find a way to export a subset of Maya joint hierarchy. I know we can use `-exportRoots` argument to `mayaUSDExport` command to export a subset of nodes from Maya to USD but this seems not working with joints exporting. When exporting with Maya joints, we need to use `-exportSkels` otherwise joints won't be exported. This works when you are exporting a whole hierarchy, however, it doesn't work if you just want to export a subset of the hierarchy which requires `-exportRoots` argument because it would error when using `-exportRoots` and `-exportSkels` together. Please see the example below. 

Given a hierarchy with a mesh and a joint:

<img width="500" alt="Screenshot 2024-02-05 at 16 44 28" src="https://github.com/Autodesk/maya-usd/assets/935278/73dd64a2-0391-4d13-9ba8-13e09977ca10">

Run the following command to export nodes that are under "parent" and "jointsGrp" group nodes. 

```python
cmds.mayaUSDExport(
    f="partialNodes.usda", sl=False, worldspace=True,  stripNamespaces=True,
    exportRoots=("parent", "jointsGrp"), shadingMode="none"
)
```
If we open the exported file In the usdview, you will see that `mayaUSDExport` command is able to export nodes that are under "parent", but for node that is under "jointsGrp", it cannot, because it's a Maya joint. 

<img width="500" alt="Screenshot 2024-02-05 at 17 03 29" src="https://github.com/Autodesk/maya-usd/assets/935278/8d1b6da3-afd8-4b30-8e0d-9b08855ed29c">

Let's try to export joints with `-exportSkels` . Run the following code to export joints: 

```python
cmds.select("parent", "jointsGrp")
cmds.mayaUSDExport(
    f="joints.usda", sl=True, worldspace=True,  stripNamespaces=True,
    shadingMode="none",
    exportSkels="auto"
)
```
If you open the exported file in usdview, you will see that joints are exported  successfully along with all of its parent nodes: 

<img width="500" alt="Screenshot 2024-02-05 at 17 08 10" src="https://github.com/Autodesk/maya-usd/assets/935278/6c3f7f8f-fff4-459e-a6ed-34fe6c35c7a0">

If we try to use `-exportSkels` and `-exportRoots` together to only export the joints under the specified nodes by running the following command: 

```python
cmds.select("parent", "jointsGrp")
cmds.mayaUSDExport(
    f="jointsUnderSpecifiedNode.usda", sl=True, worldspace=True,  stripNamespaces=True,
    exportRoots=("parent", "jointsGrp"), 
    shadingMode="none",
    exportSkels="auto"
)
```
We will get errors saying `# Error: RuntimeError: file <string> line 2: Maya command error`

**Describe the solution you'd like**
We'd like to export a subset of nodes that are Maya joints (exclude its parent nodes) under a hierarchy to USD as `Skeleton` prims. It would be great if  `-exportSkels` argument can work together with `-exportRoots` argument to achieve this. 

**Describe alternatives you've considered**
If this is not supported currently then I'd like to know if there is a plan to support this in future.

**Comment by wallworm** [comment]
Thanks for sharing. We will discuss this on the team.

**Comment by barbalt** [comment]
Closing issue as it has been addressed by #3681

**Comment by dilenshah23** [comment]
Has this been resolved?
